guides/common/modules/proc_performing-post-upgrade-tasks.adoc

```diff
@@ -5,13 +5,24 @@
 If the custom code is executed before and/or after the provisioning process, use custom provisioning snippets to avoid recreating cloned templates.
 For more information about configuring custom provisioning snippets, see {ProvisioningDocURL}Creating_Custom_Provisioning_Snippets_provisioning[Creating Custom Provisioning Snippets] in _{ProvisioningDocTitle}_.
 
+* Delete the `/var/lib/pgsql/data-old/` directory:
```

Comment:
This step is really EL specific so wrapping it in `ifndef::foreman-deb[]` is probably best.